book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+# Class representing a Book
+class Book < Item
+  attr_accessor :publisher, :cover_state, :label
```

Comment:
As  : label  (as attr_accessor) is already attr_accessor in Item, it is not important to make it  again  as attr_accessor in the child file

Reply:
[Optional]